src/firefly/js/util/WebUtil.js

```diff
@@ -351,18 +354,6 @@ export function updateSet(object, path, value) {
 }
 
 /**
- * This is a wrapper of React update's $merge for use with deep object update.
```

Comment:
I removed it thinking no one was using it.  Then I realized quickly that Emmanuel was using it.  I will add it back in.